orm_lib/src/SqlBinder.cc

```diff
@@ -266,22 +260,67 @@ SqlBinder &SqlBinder::operator<<(std::nullptr_t nullp)
     return *this;
 }
 
+SqlBinder &SqlBinder::operator<<(DefaultValue dv)
+{
+    (void)dv;
+    if (type_ == ClientType::PostgreSQL)
+    {
+        std::regex r("\\$" + std::to_string(parametersNumber_ + 1) + "\\b");
+        // initialize with empty, as the next line will make a copy anyway
+        if (!sqlPtr_)
+            sqlPtr_ = std::make_shared<std::string>();
+
+        *sqlPtr_ = std::regex_replace(sqlViewPtr_, r, "default");
+
+        // decrement all other $n parameters by 1
+        size_t i = parametersNumber_ + 2;
+        while ((sqlPtr_->find("$" + std::to_string(parametersNumber_ + i))) !=
```

Comment:
Here should be std::to_string(i) because i equals parametersNumber+1

Reply:
Yes, good catch